packages/currencies/package.json

```diff
@@ -34,7 +34,9 @@
     "flow": "flow",
     "test": "cd ../.. && export PATH=$(yarn bin):$PATH && cd - && jest",
     "clean": "rm -rf lib/ src/data/icons/react src/data/icons/reactNative",
-    "build": "node scripts/buildReactIcons.js && cd ../.. && export PATH=$(yarn bin):$PATH && cd - && babel --source-maps -d lib src && flow-copy-source -v src lib",
-    "watch": "node scripts/buildReactIcons.js && cd ../.. && export PATH=$(yarn bin):$PATH && cd - && babel --watch --source-maps -d lib src & flow-copy-source -w -v src lib"
+    "build":
+      "node scripts/buildReactIcons.js && cd ../.. && export PATH=$(yarn bin):$PATH && cd - && babel --source-maps -d lib src && flow-copy-source -v src lib",
+    "watch":
+      "node scripts/buildReactIcons.js && cd ../.. && export PATH=$(yarn bin):$PATH && cd - && babel --watch --source-maps -d lib src & flow-copy-source -w -v src lib"
   }
-}
\ No newline at end of file
+}
```

Comment:
you should ignore package.json from prettier, as this formatting will be overwritten everytime you yarn add something